components/docker/docker.go

```diff
@@ -155,7 +155,24 @@ func (d *Manager) install() (*remote.Command, error) {
 		return nil, err
 	}
 
-	return groupCmd, err
+	ecrCredsHelperInstall, err := d.host.OS.PackageManager().Ensure("amazon-ecr-credential-helper", utils.MergeOptions(d.opts, utils.PulumiDependsOn(groupCmd))...)
```

Comment:
Given that it's limited to AWS scenarios, it should not be in the common component but in `scenario/aws`.

Reply:
Oh yes, you're right. Moved.